src-core/src/spec/trajectory.rs

```diff
@@ -412,6 +412,7 @@ pub struct TrajectoryFile {
     /// Will always be in sync with the file name on disk.
     pub name: String,
     /// The version of the `.traj` file spec.
+    #[serde(default = "traj_schema_version")]
```

Comment:
I don't agree with this change, if the version isn't present its an invalid file because the upgrader doesn't know what actions to take.